extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -865,6 +865,17 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       // creative will be centered instead of left-aligned.
       this.element.removeAttribute('height');
       setStyles(this.element, {width: `${size.width}px`});
+    } else if (this.isFluidRequest_ && this.isVerifiedAmpCreative) {
+      // This is an AMP fluid creative that will be rendered in a friendly
+      // frame.
+      dev().assert(this.iframe && this.iframe.contentWindow &&
+          this.iframe.contentWindow.document &&
+          this.iframe.contentWindow.document.body,
+      'Attempting to expand fluid creative without properly set up ' +
+          'friendly frame.');
+      this.attemptChangeHeight(
+          this.iframe.contentWindow.document.body./*OK*/scrollHeight)
```

Comment:
you're sure you want htis to be body as opposed to documentElement?

Reply:
I believe so. This is what SafeFrame does.